src/test/java/org/kohsuke/github/GHOrganizationTest.java

```diff
@@ -57,4 +57,23 @@ public void testInviteUser() throws IOException {
         // Check the invitation has worked.
         // assertTrue(org.hasMember(user));
     }
+
+    @Test
+    public void testCreateTeamWithRepoAccess() throws IOException {
+        String REPO_NAME = "github-api";
+        String TEAM_NAME = "create-team-test";
+
+        GHOrganization org = gitHub.getOrganization(GITHUB_API_TEST_ORG);
+        GHRepository repo = org.getRepository(REPO_NAME);
+
+        // Clean up team if exists.
+        GHTeam existingTeam = org.getTeamByName(TEAM_NAME);
```

Comment:
This should be done in a `@Before` method using `githubBeforeAfter` connection.
https://github.com/github-api/github-api/pull/579/files#diff-ed97d1edac0ae3acc074e2e907870717R23

Reply:
Thanks, I have moved it now. Can you please check?